includes/utils/class-amp-mutation-utils.php

```diff
@@ -0,0 +1,325 @@
+<?php
+/**
+ * Class AMP_Mutation_Utils
+ *
+ * @package AMP
+ */
+
+/**
+ * Class AMP_Mutation_Utils
+ *
+ * @since 0.7
+ */
+class AMP_Mutation_Utils {
```

Comment:
What does Mutation stands for here, it could because I am native French but `AMP_Validation_Utils` would make more sense to me.

Reply:
You're tight that `AMP_Validation_Utils` would be better. Mutation refers to the `mutation_callback`, which tracks when the sanitizer removes an attribute or node.

Reply:
This [commit](https://github.com/Automattic/amp-wp/commit/f8aeca80dd2fe7589d6097b4c2a802ea6d4fb679) changes the name of the classes.